login-workflow/example/src/components/DebugComponent.tsx

```diff
@@ -43,7 +43,12 @@ export const DebugComponent = (): JSX.Element => {
                     {' '}
                     <Box sx={LinksWrapperStyles}>
                         <Typography variant="body2">
-                            <Link to={`${routes?.REGISTER_INVITE || '/register-by-invite'}`}>
+                            <Link
+                                to={`${
+                                    routes?.REGISTER_INVITE ||
+                                    '/register-by-invite?code=8k27jshInvite234Code&email=example@example.com'
```

Comment:
Any reference from where you are taking this URL or is it a random URL you are adding?

Reply:
Master branch has this code `code=DEBUG_VALIDATION_CODE_DEADBEEF` FYI.

Reply:
Updated. Please review.